mule-apikit-module/src/main/java/org/mule/module/apikit/Router.java

```diff
@@ -145,7 +145,7 @@ public IRaml getRaml() {
     URIResolver uriResolver = findInCache(path, config.getUriResolverCache());
     ResolvedVariables resolvedVariables = uriResolver.resolve(uriPattern);
 
-    IResource resource = getResource(config, attributes.getMethod().toLowerCase(), uriPattern);
+    IResource resource = config.getFlowFinder().getResource(uriPattern);
```

Comment:
Could resource be null if method is not found?

Reply:
This feature already has tests, I just moved the code from one place to another

Reply:
In APIkit context, the resource is never null. Either way, I added a check to ensure that the resource is not null. If null, a MuleRuntimeException will b thrown